spec/spec_helper.rb

```diff
@@ -42,6 +42,18 @@
 end
 
 RSpec.configure do |c|
+  # getting the correct facter version is tricky. We use facterdb as a source to mock facts
```

Comment:
Are you going to submit this to modulesync as well?

Reply:
Of course. I just need to test it here.